sway/input/seat.c

```diff
@@ -368,11 +369,23 @@ static void seat_update_capabilities(struct sway_seat *seat) {
 			caps |= WL_SEAT_CAPABILITY_TOUCH;
 			break;
 		case WLR_INPUT_DEVICE_TABLET_TOOL:
+			if (seat->cursor->proximity == WLR_TABLET_TOOL_PROXIMITY_IN) {
```

Comment:
Hmmm. This will change when we'll support the tablet protocol, not sure it's worth it to do this

Reply:
In that case, shall I just hold off with this PR until then?

Reply:
Eh, master is broken anyway because if a tablet is plugged in the seat won't advertise the POINTER capability. I'd say just add the capability if there's a tablet, no need for complicated proximity logic.

Reply:
I removed the proximity logic